### PR TITLE
V2 - enable autosizing by default

### PR DESCRIPTION
**Pull request by clausreiser**
* autosizing is now enabled by default. 
* Fields that should not autosize can be disabled by adding class `no-autosize`
* added `acf-autosize/enabledByDefault`-filter to disable this behaviour. Fields need to be enabled by adding the class `autosize` if not enabled by default. 
* sticky toolbar is only applied to autosizing fields. 
* version bump

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,12 +34,25 @@ add_filter('acf-autosize/wysiwyg/min-height', function() {
...
+
+```
+
 
 ## Usage
 - Install and activate like any other wordpress plugin
 - `[data-wysiwyg-slug]` is added to *every* wysiwyg field!
-- Add `autosize` as a class to a WYSIWYG or Textarea-Field to enable autosize for this field
-- that's basically it.
+- plugin ist enabled by default on all textareas and wysiwyg fields. You can disable it by adding the class `no-autosize` to a field.
```
Comment (moritzjacobs) [inline README.md]: *is

File: README.md [hunk 2]
```diff
@@ -34,12 +34,25 @@ add_filter('acf-autosize/wysiwyg/min-height', function() {
...
+- You can set the enabled default to false via the `acf-autosize/enabledByDefault`-filter. After that you can reenable single fields by adding the class `autosize`.
```
Comment (moritzjacobs) [inline README.md]: "the default behaviour" would be clearer

File: acf-autosize.php [hunk 3]
```diff
@@ -2,8 +2,8 @@
 /*
 Plugin Name: ACF Autosize
 Plugin URI: https://wordpress.org/plugins/acf-autosize/
-Description: A wordpress plugin to automatically resize and improve upon wysiwyg and textarea fields in Advanced Custom Fields. <strong>Please note: Version 2 will automatically apply autosize to all fields. It will be possible to deactivate autosizing via class though.</strong>
-Version: 1.3.3
+Description: A wordpress plugin to automatically resize and improve upon wysiwyg and textarea fields in Advanced Custom Fields.
+Version: 2.0.0
```
Comment (moritzjacobs) [inline acf-autosize.php:6]: Who would have thought there would be a 2.0.0 :-)

File: readme.txt [hunk 4]
```diff
@@ -20,9 +20,9 @@ A wordpress plugin to automatically resize and improve upon wysiwyg and textarea
 ## Usage
 - Install and activate like any other wordpress plugin
 - `[data-wysiwyg-slug]` is added to *every* wysiwyg field!
-- Add `autosize` as a class to a WYSIWYG or Textarea-Field to enable autosize for this field
+- plugin ist enabled by default on all textareas and wysiwyg fields. You can disable it by adding the class `no-autosize` to a field.
+- You can set the enabled default to false via the `acf-autosize/enabledByDefault`-filter. After that you can reenable single fields by adding the class `autosize`.
```
Comment (moritzjacobs) [inline readme.txt]: same as above. In the long run it would be interesting to generate the `readme.txt` automatically ⭐️

File: readme.txt [hunk 5]
```diff
@@ -36,6 +36,11 @@ Check out the source code [on Github!](https://github.com/yeah8000/acf-autosize)
 Install and activate like any other wordpress plugin
 
 == Changelog ==
+= 2.0.0 =
+* Enable by default
+* Add `acf-autosize/enabledByDefault`-filter to make disable by default
```
Comment (moritzjacobs) [inline readme.txt]: "to enable the inverse behaviour" or sth idk lol

File: src/acf-autosize-mu.hbs [hunk 6]
```diff
@@ -17,7 +17,10 @@ class ACFAutosizeMu {
 
 	public function echoJs() {
 		echo "<script type=\"text/javascript\">";
-		echo "ACFAutosize = {'wysiwyg':{'minHeight': ".apply_filters('acf-autosize/wysiwyg/min-height', 200)."}};";
+		echo "ACFAutosize = {";
+		echo "'wysiwyg':{'minHeight': ".apply_filters('acf-autosize/wysiwyg/min-height', 200)."},";
+		echo "'enabledByDefault':".apply_filters('acf-autosize/enabledByDefault', true);
+		echo "};";
```
Comment (moritzjacobs) [inline src/acf-autosize-mu.hbs:23]: 🧠

File: src/js/textarea.js [hunk 7]
```diff
@@ -1,7 +1,14 @@
...
+
+	if (ACFAutosize.enabledByDefault) {
+		textareas = $('.acf-field:not(.no-autosize) textarea')
+	} else {
+		textareas = $('.acf-field.autosize textarea')
+	}
+
 	autosize(textareas)
 
 	// safety first: update on ready and load.
```
Comment (moritzjacobs) [inline src/js/textarea.js:14]: "safety first" lol

**Review by moritzjacobs — APPROVED** [review]
LGTM, but wait with the release. I would like to do some minor improvements in a seperate PR. Merge this, then I'll go ahead.

Other than that, it looks grrrrreat 🐯

File: readme.txt (hunk 4 above)
Comment (clausreiser) [reply readme.txt]: absolutely!

File: acf-autosize.php (hunk 3 above)
Comment (clausreiser) [reply acf-autosize.php:6]: whoop!

**Pull request merged**